packages/@o3r/extractors/src/utils/localization.ts

```diff
@@ -0,0 +1,36 @@
+import * as ts from 'typescript';
+
+/**
+ * Retrieve the localization json files from TS Code
+ *
+ * @param node TSNode of the angular component class
+ * @param source Ts file source
+ */
+export function getLocalizationFileFromAngularElement(node: ts.ClassDeclaration): string[] | undefined {
+  const localizationPaths: string[] = [];
+  node.forEachChild((item) => {
+    if (!ts.isPropertyDeclaration(item)) {
+      return;
+    }
+
+    item.forEachChild((decorator) => {
+      if (
+        !ts.isDecorator(decorator)
+        || !ts.isCallExpression(decorator.expression)
+        || !ts.isIdentifier(decorator.expression.expression)
+        || decorator.expression.expression.escapedText !== 'Localization'
```

Comment:
Extract `Localization` to constant.
(The best scenario would be to use the `Localization.name` imported from `@o3r.localization` but not so easy in  `@o3r/component` except with a dynamic import and `@o3r.localization` as optional peerDep)

Reply:
This file is in @o3r/extractors so it's not possible as @o3r/localization depends on @o3r/extractors